sdk/tables/aztables/CHANGELOG.md

```diff
@@ -1,3 +1,3 @@
 # Release History
 
-## v0.1.0 (Unreleased)
+## 0.1.0 (Unreleased)
```

Comment:
I think you should be leaving the v here to align with all of our other libs, we've left it this way for the ARM libs and also Joel has a PR open for releasing synapse data-plane that also includes the v.